pkg/asset/cluster/tfvars/tfvars.go

```diff
@@ -1100,18 +1102,14 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 			Filename: TfPlatformVarsFileName,
 			Data:     data,
 		})
-
 	case nutanix.Name:
-		if rhcosImage == nil {
-			return errors.New("unable to retrieve rhcos image")
-		}
```

Comment:
Is this not needed now?

Reply:
No. static linter picked up that it is impossible for that value to be nil.